projects/plugins/boost/app/assets/src/js/features/error-notice/error-notice.module.scss

```diff
@@ -19,6 +19,10 @@
 		color: $red_50;
 	}
 
+	:global(.raw-error) {
+		color: $primary-black;
+	}
```

Comment:
Isn't `:global` a svelte thing? Does this work?

Reply:
I asked the same thing and was just as surprised about the answer :D

Reply:
On review, yep, this is working as it says on the tin. TIL. :)